app/megatron/commands/command_actions.py

```diff
@@ -364,10 +364,29 @@ def _change_pause_state(
         as_user=False
     )
     paused_word = "paused" if pause else "unpaused"
-    msg = {"text": f"Bot *{paused_word}* for user: {platform_user}."}
+    msg = {"text": f"Bot *{paused_word}* for user: {platform_user.username}."}
     channel = request_data.channel_id
     message_action = Action(
         ActionType.POST_MESSAGE, {"channel": channel, "message": msg}
     )
     integration_connection.take_action(message_action)
+
+    from_user = PlatformUser.objects.get(platform_id=request_data.user_id)
```

Comment:
@gonzalomolbar is this user different from the `PlatformUser` you pass in?

Reply:
Yeah, the passed one is the User and this one is the Costumer Service member